src/dc_receive_imf.rs

```diff
@@ -1418,7 +1444,9 @@ async fn create_or_lookup_group(
             if value == "group-avatar-changed" {
                 if let Some(avatar_action) = &mime_parser.group_avatar {
                     // this is just an explicit message containing the group-avatar,
-                    // apart from that, the group-avatar is send along with various other messages
+                    // apart from that, the group-avatar is send along with various other messages.
+                    // we do not check Param::AvatarTimestamp here:
```

Comment:
This comment makes it harder to read the code for me, not easier. I spent some time trying to understand the comment, until I looked at what the code does and realized that of course timestamp should not be checked here as we do not change the avatar.

Reply:
k, i also re-read, agreed :) seems the comment is superfluous then.